common/graphics/persistent_cache.cc

```diff
@@ -30,16 +31,17 @@ std::shared_ptr<AssetManager> PersistentCache::asset_manager_;
 std::mutex PersistentCache::instance_mutex_;
 std::unique_ptr<PersistentCache> PersistentCache::gPersistentCache;
 
-std::string PersistentCache::SkKeyToFilePath(const SkData& data) {
-  if (data.data() == nullptr || data.size() == 0) {
+std::string PersistentCache::SkKeyToFilePath(const SkData& key) {
+  if (key.data() == nullptr || key.size() == 0) {
     return "";
   }
 
-  std::string_view view(reinterpret_cast<const char*>(data.data()),
-                        data.size());
+  uint8_t sha_digest[SHA_DIGEST_LENGTH];
+  SHA1(static_cast<const uint8_t*>(key.data()), key.size(), sha_digest);
 
+  std::string_view view(reinterpret_cast<const char*>(sha_digest),
+                        SHA_DIGEST_LENGTH);
   auto encode_result = fml::Base32Encode(view);
```

Comment:
Base32 encoding the key seems unnecessary now after the digest has been calculated. Can we just convert it to a hex string using standard utilities and get rid of `fml::Base32Encode` and `Decode` altogether? Will just simplify things.

Reply:
Added a hex encode function to FML, which should be slightly faster than Base32.  Base32 is still needed for reading and writing the precompiled shader JSON format.

Reply:
Oh, right. I forgot about the JSON for training. Sounds good.